coder_sniffer/DrupalPractice/Sniffs/General/OptionsTSniff.php

```diff
@@ -81,6 +81,21 @@ public function process(File $phpcsFile, $stackPtr)
             $statementEnd = $tokens[$arrayToken]['bracket_closer'];
         }
 
+        // We want to find if the element "#options" belongs to a form element.
+        // Array with selectable options for a form element.
+        $formElements = [
+          "'checkboxes'",
+          "'radios'",
+          "'select'",
+          "'tableselect'",
+        ];
+        // Find beggining of the array containing "#options" element.
+        $startArray = $phpcsFile->findPrevious(T_EQUAL, $stackPtr, null, false, "=");
+        // Find next element on array of "#type".
+        $findType = $phpcsFile->findNext(T_CONSTANT_ENCAPSED_STRING, ($startArray + 1), null, false, "'#type'");
+        // Get the value of "#type".
+        $valueType = $phpcsFile->findNext(T_CONSTANT_ENCAPSED_STRING, ($findType + 1), null, false);
```

Comment:
$startArray, $findType and $valueType should check if the "find" results in a false, if this is the case then we can just stop checking this array as we can't determine the form element type.  Currently it will just calculate `false + 1 = 1`, after which it will start searching at the top of the document.